server/chat-commands/info.ts

```diff
@@ -2562,8 +2564,8 @@ export const commands: ChatCommands = {
 		}
 		room.settings.quotes.push({userid: user.id, quote: target, date: Date.now()});
 		room.saveSettings();
-		this.privateModAction(`${user.name} added the quote "${target}"`);
-		return this.modlog(`ADDQUOTE`, null, target);
+		this.privateModAction(`${user.name} added a new quote.`);
```

Comment:
Posting in case this gets lost in sim-dev before we resolve it; I think this should still show the added quote for ease of others viewing.